gateway-util-common/src/test/resources/log4j2-test.xml

```diff
@@ -21,15 +21,11 @@
             <PatternLayout pattern="[%-5level] %d{yyyy-MM-dd HH:mm:ss.SSS} [%t] %c{1} - %msg%n"/>
         </Console>
         <CollectAppender name="audit-forward" />
-        <JdbmStoreAndForwardAppender name="audit-store" file="target/audit" />
```

Comment:
Didn't this just change the whole behavior of the test? Wasn't this supposed to test `JdbmStoreAndForwardAppender`?

Now idk what jdbm actually does, but this change seems to completely change what the goal was.

Reply:
JdbmStoreAndForwardAppender is not used anywhere within the project, I was thinking about removing it altogether, but I can't be sure if someone uses it externally or not.

My guess that it was added to test some behaviour indirectly which later on was removed and only the test for JdbmStoreAndForwardAppender was left.

If anyone knows what was/is the purpose of this class let me know and I can put back a separate unit test for it. 

The only thing it does is storing the serialized log events in a binary file before forwarding it to an other Logger.

That's why I think maybe it was used for testing something (by recording the events and playing them back or inspecting later on). But the same log events are also stored in the log file in textual format. So I don't really understand the purpose.